js/index.js

```diff
@@ -9,5 +14,5 @@ newGameButton.addEventListener("click", () =>
 // Add 'click' event-listener to the 'Undo' button.
 const undoButton = document.getElementById("undo-move");
 undoButton.addEventListener("click", () =>
-    console.log("Undo button: Click Event Triggered")
+    BOARD.lastPieceMoved.undo(BOARD)
```

Comment:
You'll need to check if a piece has been moved before calling `undo()` so that `undo()` is not called on `null`.